packages/agent/src/agents/modes/interactive.ts

```
@@ -140,14 +166,67 @@
+        const content =
+          typeof sys.content === 'string'
+            ? sys.content
+            : JSON.stringify(sys.content);
+        systemParts.push(content);
+        idx++;
+      }
+      promptMessages.push(['system', systemParts.join('\n')]);
+
+      promptMessages.push(new MessagesPlaceholder('messages'));
+
+      const prompt = ChatPromptTemplate.fromMessages(promptMessages);
```

ESLint

[error] 'prompt' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)